src/corelibs/U2View/src/ov_mca/McaEditorStatusBar.cpp

```diff
@@ -60,10 +60,10 @@ McaEditorStatusBar::McaEditorStatusBar(McaEditor* editor,
     selectionLabel->hide();
 
     connect(editor->getSelectionController(),
-            SIGNAL(si_selectionChanged(const MaEditorSelection&, const MaEditorSelection&)),
-            SLOT(sl_update()));
+            SIGNAL(si_selectionChanged(const MaEditorSelection &, const MaEditorSelection &)),
```

Comment:
Use direct function pointers for all changed signal/slots connectors. This will guarantee that there are no wrong connections during compile time.